File: java/DifferenceVolumesCuboids.java
// find the difference in volumes of 2 cuboids

import java.util.Arrays;

public class DifferenceVolumesCuboids {

  public static int findDifference(int[] firstCuboid, int[] secondCuboid) {
    int volume1 = Arrays.stream(firstCuboid).reduce(1, (acc, cur) -> acc * cur);
    int volume2 = Arrays.stream(secondCuboid).reduce(1, (acc, cur) -> acc * cur);
    return Math.abs(volume1 - volume2);
  }

  public static void main(String[] args) {
    System.out.println(findDifference(new int[] { 3, 2, 5 }, new int[] { 1, 4, 4 })); // 14
  }
}

File: c#/DifferenceVolumesCuboids.cs
// find difference of volumes of 2 cuboids

using System;
using System.Linq;

public class DifferenceVolumesCuboids
{
	public static int FindDifference(int[] a, int[] b)
	{
		return Math.Abs(a.Aggregate(1, (a,c) => a * c) - b.Aggregate(1, (a,c) => a * c));
	}

	public static void Main(string[] args)
	{
		Console.WriteLine(FindDifference(new int[] {2,2,3},new int[] {5,4,1})); // 12
	}
}